upgrading/topics/upgrade_adapters.adoc

```diff
@@ -53,17 +55,18 @@ endif::[]
 [[_upgrade_js_adapter]]
 == Upgrading the JavaScript Adapter
 
+.Procedure
 To upgrade a JavaScript adapter that has been copied to your web application, complete the following steps: 
 
 . Download the new adapter archive.
 . Overwrite the keycloak.js file in your application with the keycloak.js file from the downloaded archive.
 
-
 [[_upgrade_nodejs_adapter]]
 == Upgrading the Node.js Adapter
 
 To upgrade a Node.js adapter that has been copied to your web application, complete the following steps:
 
+.Procedure
```

Comment:
@andymunro : Check if there should be two leading lines:
One with "following steps:" and one with "Procedure"

Reply:
Thanks, this is correct.  The procedure is introduce first and it is followed by the heading Procedure. An exception is when there are two procedures in one section. Then the heading Procedure comes first.